### ISSUE TITLE
[prql-js] Very bad template literal example 

### ISSUE THREAD

**Issue by leumasme** [PRQL/prql#4164]
### What happened?

In the [prql-js](https://www.npmjs.com/package/prql-js) bindings readme, one can find the following usage example:
```js
const prqljs = require("prql-js");
const prql = (string) => prqljs.compile(string[0] || "");

const sql = prql`from employees | select first_name`;
console.log(sql);
```
Here, a custom template literal formatter "prql" is defined and then used.  
Such template literal formatters are often seen in ORMs to handle automatically escaping variables, e.g.
```js
let username = "this_will_be_escaped"
sql`select * from users where username = ${username}`
```
the misguided use in the example was presumably picked up from such usecases and then implemented incorrectly.  
A template literal formatter will get the raw strings as an array via the first argument, then the variables to insert as spread arguments, e.g.
```js
const sql = (strings, ...variables) => // escape variables and insert them between elements of strings array
```
The example implementation simply returns the first string element. This means that if, for example
```js
prql`hello ${variable} world`
```
is called, only `hello ` will be passed to `prqljs.compile`.  

A template literal function has no purpose here in the first place, as template literals can just be passed to functions directly (where they will first get joined into a string):
```js
compile(`none
of that
nonsense`)
```
this is **not** the same as the `prql` example above; here, the compile function will simply always be called with one string (the result of the template literal).  

TLDR the template literal formatter function in the example is not only broken, but also useless and should be avoided due to the convention of template literal formatter functions preprocessing what is passed to them
 

### PRQL input

```elm
-
```

### SQL output

```SQL
-
```

### MVCE confirmation

- [X] Minimal example
- [X] New issue

### Anything else?

the error catching (example) of parsing json out of the error message is also pretty horrendous

**Comment by max-sixty** [comment]
Thanks for the issue @leumasme 

Does anyone here who knows more about JS than me have a view?

**Comment by aljazerzen** [comment]
I agree, using a template in the example is an overkill, even if `` prql`from x` `` looks cool.

@leumasme, do you care opening a PR?
